src/DUWOagent/events.mod2g

```diff
@@ -43,4 +44,15 @@ module events {
 		bel(member(indicator(ID,CurrentValue,TargetValue),X),
 		indicatorWeight(ID,_,_)) % Only insert our indicators.
 			do insert(indicator(ID,CurrentValue,TargetValue)).
+	
+	
+	%%% Adopt goals %%%
+			
+	% Adopt goal to build student housing when we need to.
+	if bel(needStudentHousing), not(goal(goalBuildStudentHousing))
+		then adopt(goalBuildStudentHousing).
+		
+	% Adopt goal to raise budget when our budget is below the target.
+	if bel(noBudget), not(goal(goalReachBudgetTarget))
+		then adopt(goalReachBudgetTarget).

```

Comment:
@klikooo This is the events module, I don't thing you're supposed to divide it into seperate modules.